apps/nextra/public/scripts/install_cli.ps1

```diff
@@ -0,0 +1,177 @@
+# This script installs the Aptos CLI on Windows.
+# It will perform the following steps:
+# - Determine the system architecture
+# - Download the CLI
+# - Put it in an appropriate location
+
+# ANSI color codes for PowerShell
+$RED = "`e[31m"
```

Comment:
```suggestion
$ESC = [char]0x1B
$RED = "${ESC}[31m"
$GREEN = "${ESC}[32m"
$YELLOW = "${ESC}[33m"
$BLUE = "${ESC}[34m"
$CYAN = "${ESC}[36m"
$BOLD = "${ESC}[1m"
$NC = "${ESC}[0m"
```

The earlier version had compatibility issues and didn't function properly in PowerShell 5.1.